lua/nvim-tree/events.lua

```diff
@@ -0,0 +1,94 @@
+local M = {}
+
+local global_handlers = {}
+
+local Event = {
+    NodeRenamed = 'NodeRenamed',
+    FileCreated = 'FileCreated',
+    FileRemoved = 'FileRemoved',
+    FolderCreated = 'FolderCreated',
+    FolderRemoved = 'FolderRemoved',
```

Comment:
could you indent this file with the `.editorconfig` settings please :) ? i use 2 spaces for indenting lua :)

Reply:
:+1: There doesn't seem to be an `.editorconfig` so I added one, let me know if you want me to remove it

Reply:
woups i thought i had one ^^' thanks for adding it